index.js

```diff
@@ -6,9 +6,12 @@ var supportsColor = require('supports-color');
 var defineProps = Object.defineProperties;
 var isSimpleWindowsTerm = process.platform === 'win32' && !/^xterm/i.test(process.env.TERM);
 
+var levelMapping = ['ansi', 'ansi', 'ansi256', 'ansi16m'];
+var skipModels = ['gray'];
+
 function Chalk(options) {
-	// detect mode if not set manually
-	this.enabled = !options || options.enabled === undefined ? supportsColor : options.enabled;
+	// detect level if not set manually
+	this.level = !options || options.level === undefined ? supportsColor.level : options.level;
```

Comment:
Because `enabled` is on by default, so it's even more weird code to determine if we default to `level = 1` or `level = supportsColor.level` based on if something is passed into the Chalk constructor.